controller/version.ml

```diff
@@ -0,0 +1,71 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *   INRIA, CNRS and contributors - Copyright 1999-2018       *)
+(* <O___,, *       (see CREDITS file for the list of authors)           *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
+
+(************************************************************************)
+(*                        UPDATE VERSION HERE                           *)
+let current_version_string = "0.1.3"
+(*                        UPDATE VERSION HERE                           *)
+(************************************************************************)
+
+type t =
+  | Version of
+      { major : int
+      ; minor : int
+      ; patch : int
+      }
+  | Any
+
+let compare_int : int -> int -> [ `Less | `Equal | `More ] =
+ fun a b -> if a < b then `Less else if a > b then `More else `Equal
+
+let compare (a : t) (b : t) =
+  match (a, b) with
+  | Any, Any -> `Equal
+  | Any, _ -> `More
+  | _, Any -> `Less
+  | Version a, Version b -> (
+    match compare_int a.major b.major with
+    | `Equal -> (
+      match compare_int a.minor b.minor with
+      | `Equal -> compare_int a.patch b.patch
+      | x -> x)
+    | x -> x)
```

Comment:
That seems pretty over engineered to me, moreover, I don't want to maintain this code here, I'd rather use `semver2` if we ever have a need.

Let's keep things simple for now.